proto/v1/review_service.proto

```diff
@@ -177,6 +177,7 @@ message ApprovalTemplate {
   ApprovalFlow flow = 1;
   string title = 2;
   string description = 3;
+  int32 creator_id = 4 [(google.api.field_behavior) = OUTPUT_ONLY];
```

Comment:
We'd like to get rid of the int format in v1. Better us string (users/a@bytebase.com).

Reply:
Updated